crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -67,13 +66,21 @@ impl Default for ReactJsxOptions {
     }
 }
 
-#[derive(Debug, Default, Clone, Copy, Deserialize)]
+#[derive(Deserialize, Debug, Clone)]
+#[serde(untagged)]
+pub enum ReactJsxRuntimeOption {
+    Valid(ReactJsxRuntime),
+    // The order matters. The most permissive variant (i.e. the catch-all)
+    // should be tried last during deserialization.
+    Unknown(String),
+}
```

Comment:
What about adding a `Default` tag here instead of a confusing `Option<ReactJsxRuntimeOption>`?

Reply:
Do you mean 
```rs
pub struct ReactJsxOptions {
  ...
   runtime: Option<ReactJsxRuntime>
}
```
This data structure did work, cause any invalid or empty input of runtime will be deserialized to `None`, 
we need to figure out if the user passes an invalid runtime, and give some waring to pass a babel snapshot, and fallback to a default runtime if `runtime` is empty.